config/samples/storage_v1_csm_powerflex.yaml

```diff
@@ -55,7 +55,7 @@ spec:
       - name: snapshotter
         image: registry.k8s.io/sig-storage/csi-snapshotter:v8.1.0
       - name: csi-metadata-retriever
-        image: quay.io/dell/container-storage-modulescsi-metadata-retriever:v1.8.0
+        image: quay.io/dell/container-storage-modules/csi-metadata-retriever:v1.8.0
```

Comment:
Can we check if there is a nightly image for this also?

Reply:
Covered  here : https://github.com/dell/csm-operator/pull/762